README.md

```diff
@@ -16,24 +19,43 @@ practices with Custom Resources and Kubernetes-native primitives.
 
 ---
 
-[Documentation](#documentation) | [Learning with Videos](#learning-with-videos) | [Quickstart Examples](#quickstart-examples)
-| [IDE Integration](#ide-integration) | [Community](#community)
+[Project Status](#project-status) | [Introduction](#introduction) | [Documentation](#documentation) | [Learning with Videos](#learning-with-videos) | [Quickstart Examples](#quickstart-examples) | [Community](#community)
 
 ---
 
-## Documentation
+## Project Status
+Tyk Operator is currently in beta. We follow the [Kubernetes definition of beta](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/#feature-stages). We welcome users to try it out and give us feedback [here](https://github.com/TykTechnologies/tyk-operator/discussions) or via support channel for existing Tyk customers.
+
+
+## Introduction
+
+### What can you do with Tyk Operator?
 
-Read more about the [concepts here](./docs/concepts.md).
+Tyk Operator can configure Tyk Gateway as a drop-in replacement for standard [Kubernetes Ingress](./docs/ingress.md). You can manage your API definitions and security policies with it. It also works with the classic portal so you can manage your Classic Portal declaratively.
 
-Custom Tyk Objects are available as [CRDs](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/)
-and documentation for each of these custom resources are available:
+Custom Tyk Objects are available as [CRDs](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) and documentation for each of these custom resources are available [here](./docs/crd.md).
```

Comment:
There is no `./docs/crd.md` file. Did you mean [`./docs/api_definitions.md`](https://github.com/TykTechnologies/tyk-operator/blob/caroltyk-readme-update/docs/api_definitions.md) file?

```suggestion
Custom Tyk Objects are available as [CRDs](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) and documentation for each of these custom resources are available [here](./docs/api_definitions.md).
```

Reply:
I'll be adding a crd.md page